Add Content surrounding serializable values (#1182)
withastro/docs: src/pages/en/reference/directives-reference.md

### `define:vars`

> `define:vars={...}` can pass server-side variables from your component front matter into the client `<script>` or `<style>`. Any *serializable* front matter variable is supported, such as; `strings`, `numbers`,`object`, `array`, including `props` passed to your component through `Astro.props`.

Reviewer: ```suggestion
`define:vars={...}` can pass server-side variables from your component front matter into the client `<script>` or `<style>`. Any JSON-serializable front matter variable is supported, including `props` passed to your component through `Astro.props`. Values are serialized with [`JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).
```

> `define:vars={...}` can pass server-side variables from your component front matter into the client `<script>` or `<style>`. Any JSON-serializable front matter variable is supported, including `props` passed to your component through `Astro.props`. Values are serialized with [`JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).

Reviewer 2: ```suggestion
`define:vars={...}` can pass server-side variables from your component frontmatter into the client `<script>` or `<style>` tags. Any JSON-serializable front matter variable is supported, including `props` passed to your component through `Astro.props`. Values are serialized with [`JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).
```
